pynest/nest/ll_api.py

```diff
@@ -398,8 +399,17 @@ def init(argv):
         else:
             try:
                 import keyword
-                from .lib.hl_api_models import Models		# noqa
-                keyword.kwlist += Models()
+                from .lib.hl_api_simulation import GetKernelStatus  #noqa
+                keyword_lists = (
+                    "connection_rules",
+                    "node_models",
+                    "recording_backends",
+                    "rng_types",
+                    "stimulation_backends",
+                    "synapse_models",
+                )
+                for kwl in keyword_lists:
+                    keyword.kwlist += GetKernelStatus(kwl)
             except ImportError:
                 pass
```

Comment:
Why continue if `from .lib.hl_api_simulation import GetKernelStatus` fails? Doesn't that point to a non-working PyNEST installation and we could quit at once?

Reply:
I'm not the original author of the code, but think that the `except` is meant to catch a missing `keyword` module rather than to catch the `import` that you pointed out. If my interpretation is right, I think the code is fine as is.

Reply:
But `keyword` has been part of the Python library since at least 2.7 (see https://docs.python.org/2.7/library/keyword.html), so it should always be there.

Reply:
Alright. I've moved the keyword import in 36ac587c1.